test/ut/retiarii/test_convert_operators.py

```diff
@@ -1387,5 +1387,29 @@ def forward(self, x):
         x = torch.randn(1, 2, 3, 1, requires_grad=False).int()
         self.checkExportImport(SimpleOp(), (x, ))
 
+    def test_basic_instancenorm_2d(self):
+        class SimpleOp(nn.Module):
+            def __init__(self):
+                super().__init__()
+                
+            def forward(self, x):
+                out = torch.nn.InstanceNorm2d(x)
+                return out
+
+        x = torch.randn(2, 2, 3, 1, requires_grad=False).int()
+        self.checkExportImport(SimpleOp(), (x,))
+
+    def test_basic_pixelshuffle(self):
+        class SimpleOp(nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, x):
+                out = torch.nn.PixelShuffle(x)
+                return out
+
+        x = torch.randn(4, 9, 4, 4, requires_grad=False).int()
+        self.checkExportImport(SimpleOp(), (x,))
```

Comment:
the unittest should be put in `test_model_speedup.py` and test whether speedup works for these two ops.